helm/cluster-shared/templates/_coredns.tpl

```diff
@@ -63,121 +65,6 @@ data:
       labels:
         {{- include "labels.common" . | nindent 8 }}
     ---
-{{- if .Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy" }}
```

Comment:
I'd rather we keep this. The `privileged` PSP gives more capabilities than is needed by CoreDNS (e.g. `hostNetwork`, `hostIPC` and `hostPID`)

Reply:
And if I'm not mistaken, you're not actually attaching the privileged PSP to the coredns role in this PR?

Reply:
@bavarianbidi This line is still needed isn't it?